### PR TITLE
Initial governance model

### PR DESCRIPTION
**Pull request by mattrosno**
Closes carbon-design-system/issue-tracking#83.

#### To do

- [x] Incorporate "projects" across site instead of list of repos? (Future effort.)
- [x] Include board of directors function and members? (Future effort.)
- [X] Review design contribution guidelines (`carbon-contributions` repo, `design-system-website` references)? Discussed with Anna - she'll resolve.
- [x] Review developer contribution guidelines
- [x] Review add-on contribution guidelines
- [x] Review help page
- [X] Need to site sources / inspiration for governance model content? Nope we good.
- [X] Pattern for listing contributors & maintainers & lead maintainer in each repo
- [x] Document maintainers and lead maintainer in each repo (Future effort.)
- [x] Document issue and PR labels in each repo (Future effort.)
- [x] Make sure each repo has code of conduct, etc. (Future effort.)

### CODE REVIEW

**Comment by joshblack** [comment]
Some thoughts around the project teams and how we could organize them:

- Core: responsible for spec, steering body of the project
- Components: responsible for implementation of appearance and behavior
  - Sub-teams:
    - Angular
    - React
    - Styles
    - Vanilla
    - Vue
- Elements: responsible for implementing the IDL
- Design Kit: responsible for design kits
- Website: responsible for online presence

**Comment by cal-smith** [comment]
Hey 👋 this looks pretty great! Since this is adopting a TSB/TSC style maybe some clarity on how the "core team" and the TSB/TSC is determined would be useful?

The current (as of my typing) "Teams" section adds some confusion - I think the structure @joshblack proposed cleans that up a bit. Makes it clear that the Core team isn't _necessarily_ the same as the carbon-components and carbon-components-react team.

I'd expect each team lead (including sub-team leads) would be required to participate in TSC discussions - true?

**Comment by mattrosno** [comment]
@joshblack @cal-smith thanks for the feedback on team structure definitions. I'll iterate.

I intentionally left TSC pretty "blank canvas" for the time being, until that's better defined / up and running. But yes, the expectation would be for each team lead to have a seat at the table for two-way communication.

**Comment by cal-smith** [comment]
Looks solid to me 👍 As you said most of the TSC details will be worked out as we start running it.

I'll bow out for the other notes (but the 🚧 and 👑 emoji make sense to me)

**Comment by claycrenshaw** [comment]
A couple of thoughts:

First, thank you for creating this! There is a lot of great info in here!

Is "governance" definitely the right title for this page? Will users outside of the Carbon team understand what content to expect from that title? Would something like "contribution model" be more descriptive and less ambiguous? No strong feeling personally on this one, just asking.

Good stuff, Matt!

**Comment by claycrenshaw** [comment]
Made some suggested edits :)

File: src/content/contributing/governance/governance.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+Project teams are narrowly focused on specific areas of interest. Although not required, it's recommended that project teams include at least one member of the core team to help ensure coherent project vision. Under the distributed model, project teams operate independently, though it's recommended that they follow this governance structure to define roles, responsibilities, and decision-making protocols.
+
+Each project team has a lead maintainer, responsible for:
+
+- Participating in the [technical steering committee](#technical-steering-committee)
+- Deciding on the initial membership of project maintainers (in consultation with the core team)
+- Determining and publishing project team policies and mechanics, including the way maintainers join and leave the team (which should be based on team consensus)
+- Communicating core vision to the team
+- Ensuring that issues and pull requests progress at a reasonable rate, assigning issue owners and pull request reviewers as necessary
```
Comment (asudoh) [inline src/content/contributing/governance/governance.md]:
```suggestion
- Act as a point of primary contact for the team
- Ensuring that issues and pull requests progress at a reasonable rate, assigning issue owners and pull request reviewers as necessary
```

File: src/content/contributing/governance/governance.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+- [Carbon Icons](https://github.com/IBM/carbon-icons)
+- [Carbon Elements](https://github.com/IBM/carbon-elements)
+
+**Components** - component implementation, appearance, and behavior
+
+<!-- Remove comment in v11 -->
+<!-- - [Carbon Spec and Styles](https://github.com/carbon-design-system/carbon-spec) -->
+- [Carbon Components Vanilla](https://github.com/IBM/carbon-components)
+- [Carbon Components React](carbon-components-react)
```
Comment (scottnath) [inline src/content/contributing/governance/governance.md]: `[Carbon Components React](https://github.com/IBM/carbon-components-react)`

**Pull request merged**